ai_agents/agents/ten_packages/extension/elevenlabs_tts2_python/extension.py

```
@@ -395,73 +394,64 @@ async def request_tts(self, t: TTSTextInput) -> None:
+                    module=ModuleType.TTS,
+                    code=ModuleErrorCode.FATAL_ERROR,
+                    vendor_info={"vendor": "elevenlabs"},
+                ),
+            )
+            return
+
+        try:
+            # Cancel current connection (maintain original flush disconnect behavior)
+            self.client.cancel()
+            await self.handle_completed_request(TTSAudioEndReason.INTERRUPTED)
+        except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)